src/ucp/tag/offload.c

```diff
@@ -201,15 +222,15 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_tag_offload_unexp_rndv,
     if (remote_addr) {
         /* Unexpected tag offload RNDV */
         ucp_tag_offload_fill_rts(rts, rndv_hdr, stag, remote_addr, length, 0);
-        len = ucp_tag_offload_copy_rkey(rts, (void*)rkey_buf, rkey_size) +
+        len = ucp_tag_offload_copy_rkey(worker->context, rts, (void*)rkey_buf, rkey_size) +
```

Comment:
it is incorrect, since rts allocated with `sizeof(*rts) + rkey_size` length and you need more memory in `ucp_tag_offload_copy_rkey`